.vscode/settings.json

```diff
@@ -1,5 +1,6 @@
 {
   "cSpell.words": [
+    "preve",
```

Comment:
Is vscode or a plugin nagging about spelling?

Reply:
This is a spell check extension.